.circleci/config.yml

```diff
@@ -103,6 +103,9 @@ jobs:
       - run:
           name: "Running gas cost tests"
           command: yarn run test:contracts:gasCosts
+      - run:
+          name: "Running gas cost diff check"
+          command: npx codechecks
```

Comment:
Why `npx` and not our default `yarn`?

Reply:
I'm fairly certain that codechecks doesn't run separate gas cost tests, but rather uses the artifacts from the previous run to output a comparison -- note that codechecks runs in 5 seconds, which is definitely not enough time for the gas cost tests.

I used `npx` because that was recommended, also because codechecks doesn't run locally but only in Circle, so creating a `yarn` command in `package.json` seemed unnecessary, although I can do that if you feel it's important for consistency.

Reply:
I can try running codechecks without our separate gas tests to be sure though.

Reply:
I think it still needs whatever `eth-gas-reporter` churns out. That reporter is only set in the gas-cost-tests. But yes we can use `yarn run codechecks` instead of `npx` for consistency.

Reply:
Yup we still need to run the regular gas tests: https://circleci.com/gh/JoinColony/colonyNetwork/9213

I'll update the invocation.

Reply:
You didn't have to add `circleci:codechecks` in package.json, you can just call `yarn run codechecks` from within `.circleci/config.yml`

Reply:
Yup you are right.